pkg/new-ui/v1beta1/frontend/src/app/constants/algorithms-settings.const.ts

```diff
@@ -89,6 +89,12 @@ export const CMAESSettings: AlgorithmSetting[] = [
     value: 0.001,
     type: AlgorithmSettingType.FLOAT,
   },
+  {
+    name: 'restart_strategy',
+    value: 'none',
+    values: ['none', 'ipop', 'bipop'],
+    type: AlgorithmSettingType.STRING,
+  },
```

Comment:
I checked the new Katib UI.

<img width="1481" alt="ScreenShot 2021-04-23 21 15 12" src="https://user-images.githubusercontent.com/5564044/115869712-3e690480-a479-11eb-8037-2f43e2b49c86.png">